utils/build/build.js

```diff
@@ -275,6 +275,22 @@ for (const bundle of bundles) {
   });
 }
 
+// Build/watch trace viewer service worker.
+steps.push({
+  command: 'npx',
+  args: [
+    'vite',
+    '--config',
+    'vite.sw.config.ts',
+    'build',
+    ...(watchMode ? ['--watch', '--minify=false'] : []),
+    ...(withSourceMaps ? ['--sourcemap=inline'] : []),
+  ],
+  shell: true,
+  cwd: path.join(__dirname, '..', '..', 'packages', 'trace-viewer'),
+  concurrent: watchMode, // feeds into trace-viewer's `public` directory, so it needs to be finished before trace-viewer build starts
```

Comment:
I moved this block up so it mirrors ordering. this line is the only change though.